generative_ai/context_caching/update_context_cache.py

```diff
@@ -11,28 +11,36 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import os
 
+PROJECT_ID = os.getenv("GOOGLE_CLOUD_PROJECT")
 
-def update_context_cache(project_id: str, cache_id: str) -> str:
+
+def update_context_cache(cache_id: str) -> str:
     # [START generativeaionvertexai_gemini_update_context_cache]
     import vertexai
     import datetime
 
     from vertexai.preview import caching
 
     # TODO(developer): Update and un-comment below lines
-    # project_id = "PROJECT_ID"
-    # cache_id = "CACHE_ID"
+    # PROJECT_ID = "your-project-id"
+    # cache_id = "your-cache-id"
 
-    vertexai.init(project=project_id, location="us-central1")
+    vertexai.init(project=PROJECT_ID, location="us-central1")
 
     cached_content = caching.CachedContent(cached_content_name=cache_id)
 
     # Update the expiration time by 1 hour
     cached_content.update(ttl=datetime.timedelta(hours=1))
-
     cached_content.refresh()
+
     print(cached_content.expire_time)
-    # [END generativeaionvertexai_gemini_update_context_cache]
 
     return cached_content.expire_time
+
+
+# [END generativeaionvertexai_gemini_update_context_cache]
```

Comment:
Please move the region tag back to its initial location & also add example reponse

Reply:
Done✅